app/models/facility.rb

```diff
@@ -3,25 +3,28 @@
 class Facility < ApplicationRecord
   include Mergeable
   include QuarterHelper
-  extend FriendlyId
 
-  attribute :import, :boolean, default: false
-  attribute :organization_name, :string
-  attribute :facility_group_name, :string
-
-  belongs_to :facility_group, optional: true
+  extend FriendlyId
```

Comment:
This entire file looks like it was just reorganized? The diff makes it hard to see if there were any logic changes. Let's try and keep reformatting to different PRs.

Reply:
Reverted. The formatting in this file was not sensible in any way, but it makes sense to keep this re-organization separate. 👍